innovation.game.php

```diff
@@ -5514,6 +5519,8 @@ function argSelectionMove() {
 
             // id 122, Artifacts age 1: Mask of Warka
             case "122N1A":
+            // id 157, Artifacts age 5: Bill of Rights
```

Comment:
In order to be consistent with the rest of our code (and to keep the cards in order), I split this into two separate cases, even though it results in a little bit of extra duplicated code.